pkg/http/api_test.go

```diff
@@ -1208,12 +1208,25 @@ func (f *fakeHTTPServer) getRouter(endpoints []Endpoint) *routing.Router {
 	router := routing.New()
 
 	for _, e := range endpoints {
-		methods := strings.Join(e.Methods, ",")
 		path := fmt.Sprintf("/%s/%s", e.Version, e.Route)
-
-		router.To(methods, path, e.Handler)
+		for _, m := range e.Methods {
+			if m == http.MethodGet {
```

Comment:
We can use one-line [Handle](https://github.com/fasthttp/router/blob/7ef9a432b73c86c9d15eab41d52de2eaf74f7da7/router.go#L240) instead of nested ifs..

```
router.Handle(m, path, e.Handler)
```

Reply:
Changed